classes/controller/AdminController.php

```diff
@@ -4166,8 +4166,21 @@ protected function processBulkStatusSelection($status)
                 /** @var ObjectModel $object */
                 $object = new $this->className((int) $id);
                 $object->active = (int) $status;
-                $result &= $object->update();
+                $isUpdated = (bool) $object->update();
+                $result &= $isUpdated;
+
+                if (!$isUpdated) {
+                    $this->errors[] = $this->trans('Can\'t update #%id% status', ['%id%' => (int) $id], 'Admin.Notifications.Error');
```

Comment:
```suggestion
                    $this->errors[] = $this->trans('#%id% status cannot be updated.' => (int) $id], 'Admin.Notifications.Error');
```

It would be better to tell the user why it can't be updated and how to solve the issue. Could you tell me more about it so I can come up with a new wording?

Reply:
This message is sent when the reason is unknown ... When the reason is known the message is managed in another place.
Here is an example when I try to change the status of an invalid carrier:
![image](https://user-images.githubusercontent.com/16455155/97005038-26eaa680-153e-11eb-97a8-00faaa8c6aa7.png)

Reply:
Thank you @PululuK 

@LouiseBonnard What should we do in that case? Should we keep "#%id% status cannot be updated." ?

Reply:
It suits me fine!

Reply:
@Julievrz  if it is ok for you you can remove `waiting for wording` label please ? Thanks !